cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -76,6 +81,13 @@ var (
 	kubeAPIQPS   = flag.Float32("kube-api-qps", 5, "QPS to use while communicating with the kubernetes apiserver. Defaults to 5.0.")
 	kubeAPIBurst = flag.Int("kube-api-burst", 10, "Burst to use while communicating with the kubernetes apiserver. Defaults to 10.")
 
+	capacityFeatures = func() *capacity.Features {
```

Comment:
I wanted to keep all argument definitions at the top and missed that there was already one being defined further below. I'd rather move `feature-gates` up instead of `capacity-features` down into main, but I don't feed strongly about this.

Reply:
ack